operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -514,6 +515,31 @@ function createModelDomain() {
   trace "Exiting createModelDomain"
 }
 
+
+# Expands into the root directory the MII domain configuration, stored in one or more config maps
+function restoreDomainConfig() {
+  restoreEncodedTar "domainzip.secure" || return 1
+
+  chmod +x ${DOMAIN_HOME}/bin/*.sh ${DOMAIN_HOME}/*.sh  || return 1
+}
+
+# Expands into the root directory the MII primordial domain, stored in one or more config maps
+function restorePrimordialDomain() {
+  restoreEncodedTar "primordial_domainzip.secure" || return 1
+}
+
+# Restores the specified directory, targz'ed and stored in one or more config maps after base 64 encoding
+# args:
+# $1 the name of the encoded file in the config map
+function restoreEncodedTar() {
+  cd / || return 1
+  cat ${OPERATOR_ROOT}/introspector*/${1} > /tmp/domain.secure
+  base64 -d "/tmp/domain.secure" > /tmp/domain.tar.gz || return 1
+
+  tar -xzf /tmp/domain.tar.gz || return 1
```

Comment:
@jshum2479 This is unrelated to Russ' enhancement, but shouldn't the old /tmp file be removed upon success? I assume it takes up a good amount of space.   

```
tar -xzf /tmp/domain.tar.gz || return 1
rm /tmp/domain.tar.gz 
```

Reply:
modelInImage only affects the introspect pod which will go away anyway,  but since this method will be called in startServer.sh, we should clean up the specific file.